### PR TITLE
update notes text box

### PR DESCRIPTION
**Pull request by gavinkalika**
update notes text box

- This time worked on updated version of master.

Please reject the **other** pull request by me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -203,8 +203,9 @@ <h2 class="bannerbox__title">I need a ride</h2>
                                 </label>
                             </div>
                             <div class="form-group">
-                                <label for="RiderAccommodationNotes">Notes <i class="optional">Optional</i></label>
-                                <textarea class="form-input" id="RiderAccommodationNotes" placeholder="Accommodation requirements e.g. service animal, assistance folding equipment, assistance entering/exiting the vehicle, etc." name="RiderAccommodationNotes" rows="4"></textarea>
+                                <label for="RiderAccommodationNotes">Notes</label>
```
Comment (richardwestenra) [inline index.html]: Hey, why did you remove the _Optional_ label? That wasn't part of the task description. Can you please restore it?

**Comment by gavinkalika** [comment]
@richardwestenra added the optional label back. sorry about that!

**Comment by richardwestenra** [comment]
Nice, thanks! LGTM :)

**Comment by gavinkalika** [comment]
ya! first pr!

**Comment by richardwestenra** [comment]
Congrats! Nice work :)
